patches/minecraft/net/minecraft/client/model/HumanoidModel.java.patch

```diff
@@ -0,0 +1,29 @@
+--- a/net/minecraft/client/model/HumanoidModel.java
++++ b/net/minecraft/client/model/HumanoidModel.java
+@@ -19,7 +_,8 @@
+ import net.minecraftforge.api.distmarker.OnlyIn;
+ 
+ @OnlyIn(Dist.CLIENT)
+-public class HumanoidModel<T extends LivingEntity> extends AgeableListModel<T> implements ArmedModel, HeadedModel {
++public class HumanoidModel<T extends LivingEntity> extends AgeableListModel<T> implements ArmedModel, HeadedModel, net.minecraftforge.client.extensions.IForgeEntityModel<T>
++{
```

Comment:
In Minecraft classes (those which patches are generated for), the code style of the class should be followed rather than Forge's code style: 3 tabs, braces on the same line.